crates/iota-protocol-config/src/lib.rs

```diff
@@ -1640,7 +1648,10 @@ impl ProtocolConfig {
                 1 => unreachable!(),
                 // version 2 is a new framework version but with no config changes
                 2 => {}
-                3 => {}
+                // version 3
+                3 => {
+                    cfg.feature_flags.relocate_event_module = true;
```

Comment:
You should mention this change in the comments where it currently says
`// Version 3: TODO`

Reply:
Got it. Thanks. Added the comments in [62dcf57](https://github.com/iotaledger/iota/pull/4712/commits/62dcf57341d5412966740d0f5b5b703c9b47f2a3).